integration-tests/tests/analysis-context-bar.js

```diff
@@ -15,16 +15,26 @@ const testAnalysisContextBarFn = _.flow(
   // Navigate to appropriate part of UI (the analysis tab)
   await performAnalysisTabSetup(page, token, testUrl, workspaceName)
 
+  // Ensure UI displays the runtime Terminal icon is present + disabled
+  const tooltipTextTerminal = 'Terminal'
+  await findElement(page, clickable({ textContains: tooltipTextTerminal, isEnabled: false }))
+
   // Create a runtime
   await click(page, clickable({ textContains: 'Environment Configuration' }))
   await findElement(page, getAnimatedDrawer('Cloud Environment Details'))
   await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Settings' })) })
   await findElement(page, getAnimatedDrawer('Jupyter Cloud Environment'), { timeout: 40000 })
   await noSpinnersAfter(page, { action: () => click(page, clickable({ text: 'Create' })) })
 
-  // Ensure UI displays the runtime is creating and the terminal icon is present + disabled
-  await findElement(page, clickable({ textContains: 'Terminal', isEnabled: false }))
-  await click(page, clickable({ textContains: 'Jupyter Environment ( Creating )' }), { timeout: 40000 })
+  // For page synchronization, find the Start button
+  await findElement(page, clickable({ textContains: 'Start' }), { visible: true })
```

Comment:
What Start button is this? Could you send a screenshot of it?

Reply:
![1](https://user-images.githubusercontent.com/35533885/172490815-bcd9fed6-3f08-4541-a5da-3316d8e2ca75.jpg)

Reply:
Thank you.

Are you saying that without the line

```javascript
await findElement(page, clickable({ textContains: 'Start' }), { visible: true })
```

```javascript
await findElement(page, clickable({ textContains: tooltipTextTerminal, isEnabled: true }), { visible: true })
```

executes too soon causing failures?

Reply:
Clarified offline. No action necessary.